fuse_core/test/test_local_parameterization.cpp

```diff
@@ -31,37 +31,61 @@
  *  ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  *  POSSIBILITY OF SUCH DAMAGE.
  */
+#include <fuse_core/ceres_macros.h>
+
+#if CERES_VERSION_AT_LEAST(2, 1, 0)
+#include <ceres/autodiff_manifold.h>
+#else
 #include <fuse_core/autodiff_local_parameterization.h>
+#endif
 #include <fuse_core/eigen.h>
 #include <fuse_core/eigen_gtest.h>
 
 #include <gtest/gtest.h>
 
 
+#if CERES_VERSION_AT_LEAST(2, 1, 0)
+struct TestFunctor
+{
+  template<typename T>
+  bool Plus(const T* x, const T* delta, T* x_plus_delta) const
+#else
 struct Plus
 {
   template<typename T>
   bool operator()(const T* x, const T* delta, T* x_plus_delta) const
+#endif
   {
     x_plus_delta[0] = x[0] + 2.0 * delta[0];
     x_plus_delta[1] = x[1] + 5.0 * delta[1];
     x_plus_delta[2] = x[2];
     return true;
   }
+#if CERES_VERSION_AT_LEAST(2, 1, 0)
+
+  template<typename T>
+  bool Minus(const T* y, const T* x, T* y_minus_x) const
+#else
 };
 
 struct Minus
 {
   template<typename T>
-  bool operator()(const T* x1, const T* x2, T* delta) const
+  bool operator()(const T* y, const T* x, T* y_minus_x) const
+#endif
   {
-    delta[0] = (x2[0] - x1[0]) / 2.0;
-    delta[1] = (x2[1] - x1[1]) / 5.0;
+    y_minus_x[0] = (x[0] - y[0]) / 2.0;
```

Comment:
```suggestion
    y_minus_x[0] = (y[0] - x[0]) / 2.0;
```
isn't this y - x?
Also not sure what the 2 and 5 here are doing :thinking: